site/content/docs/5.1/components/navs-tabs.md

```diff
@@ -343,6 +343,9 @@ Use the tab JavaScript plugin—include it individually or through the compiled
     <li class="nav-item" role="presentation">
       <button class="nav-link" id="contact-tab" data-bs-toggle="tab" data-bs-target="#contact" type="button" role="tab" aria-controls="contact" aria-selected="false">Contact</button>
     </li>
+    <li class="nav-item" role="presentation">
+      <button class="nav-link disabled" id="disabled-tab" data-bs-toggle="tab" data-bs-target="#disabled" type="button" role="tab" aria-controls="disabled" aria-selected="false">Disabled</button>
```

Comment:
would be more clear to differentiate the id name a bit more (ex: disabled-nav-link, disabled-tab etc)

Reply:
I've made a change in https://github.com/twbs/bootstrap/pull/36169/commits/1ba07ee0a8d498e173e92aad2f1df04ce6b42d6c only for the first example in order to avoid having `target="#disabled"` and `aria-controls="disabled"` that can be indeed confusing.